lib/utils.ts

```diff
@@ -86,4 +84,8 @@ export function getFolders(
 		baselineFolder: (methodOptions.baselineFolder ? methodOptions.baselineFolder : folders.baselineFolder),
 		diffFolder: (methodOptions.diffFolder ? methodOptions.diffFolder : folders.diffFolder)
 	};
-}
\ No newline at end of file
+}
+
```

Comment:
Can you please add docs for the method, to be consistent with the rest?

Reply:
Is it really need create new property? I think it's redundant cuz there is still backward compatibility. Now, `baselineFolder` property can be string, function or async function. (I checked this)